setup.py

```diff
@@ -58,7 +58,7 @@
     "treelib~=1.6",
     "packaging"
 ]
-EXTRAS = {"uamqp": ["uamqp~=1.2"]}
+EXTRAS = {"uamqp": ["uamqp>=1.2, <=1.6.8"]}
```

Comment:
nit, spacing, not sure it matters but to keep consistent with dev_requirements:
```suggestion
EXTRAS = {"uamqp": ["uamqp>=1.2,<=1.6.8"]}
```